02-analytics/technical.md

```diff
@@ -0,0 +1,377 @@
+
+### Data
+
+| Data          | Name         | Commnets                                      |
+|---------------|--------------|-----------------------------------------------|
+| Event ID      | event        | Hardcoded: `playClick`                        |
+| Page Name     | pageName     | same ID as for page/screen tracking           |
+| Element Type  | elementType  | episode, series, ...                          |
+| Element ID    | elementId    | series, episode ID                            |
+
+## Search (search)
+
+*When*: When user performs a search
+
+*Reason*: Spotting issues, Usage info
+
+### API
+
+Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-specification/rudderstack-spec/track
+
+### Data
+
+| Data           | Name              | Commnets                     |
+|----------------|-------------------|------------------------------|
+| Event ID       | event             | Hardcoded: `search`          |
+| Search Term    | searchText        | exact search term            |
+| Search Latency | searchLatency     | how long did the search take |
+| Result Count   | searchResultCount |                              |
+
+
+## Search Result click (searchClick)
+
+*When*: When user clicks/taps on a search result
+
+*Reason*: Spotting issues, Usage info
+
+### API
+
+Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-specification/rudderstack-spec/track
+
+### Data
+
+| Data            | Name                 | Commnets                     |
+|-----------------|----------------------|------------------------------|
+| Event ID        | event                | Hardcoded: `searchClick`     |
+| Search Term     | searchText           | exact search term            |
+| Result Position | searchResultPosition | how long did the search take |
+| Element Type    | elementType          | episode, series, ...         |
+| Element ID      | elementId            | series, episode ID           |
+
+## Language Change (languageChange)
+
+*When*: When user changes the language in any context
+
+*Reason*: Spotting issues, Usage info
+
+*Notes*: This is not high priority, if hard to get, can be omitted
+
+### API
+
+Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-specification/rudderstack-spec/track
+
+### Data
+
+| Data          | Name               | Commnets                            |
+|---------------|--------------------|-------------------------------------|
+| Event ID      | event              | Hardcoded: `languageChange`         |
+| Page Name     | pageName           | same ID as for page/screen tracking |
+| From Language | languageFrom       |                                     |
+| To Language   | languageTo         |                                     |
+| Where         | languageChangeType | audio, app, subs                    |
+
+## Share (share)
+
+*When*: When user changes the language in any context
```

Comment:
Copy paste mistake

Reply:
@michalbcc you did a review on an outdated version of the document...